packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -374,6 +377,28 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
       />
     );
   }, [fileSharing?.accept, fileSharing?.multiple, fileSharing?.uploadHandler, fileUploadButtonOnChange]);
+
+  /* @conditional-compile-remove(file-sharing) */
+  const activeFileUploads = useSelector(fileUploadsSelector).files;
+
+  const SelectedSendBox = (): JSX.Element => {
+    /* @conditional-compile-remove(rich-text-editor) */
+    if (options?.richTextEditor !== false) {
+      return <RichTextSendBox {...richTextSendBoxProps} />;
```

Comment:
We should load the rich text send box lazyly here otherwise Contoso will always get all rooster JS dependencies.
https://react.dev/reference/react/lazy

Reply:
Let me merge lazy loading branch into this.